README.md

```diff
@@ -9,5 +9,7 @@ More concretely, each folder contains:
 - `khiva-python`: with the Khiva library and its Python binding installed.
 - `khiva-java`: with the Khiva library and its Java binding installed.
 - `khiva-r`: with the Khiva library and its R binding installed.
+- `khiva-ci-build`: the main purpose of this image is generating the Khiva installer for Travis CI.
```

Comment:
.... : prepares a Docker image with the Khiva dependencies satisfied to generate the Khiva installer for Travis CI.